.github/mergify.yml

```diff
@@ -8,6 +30,7 @@ pull_request_rules:
     - base=main
     - label!=hold
     - label!=do-not-merge
+    - label!=needs-rebase
```

Comment:
This doesn't hurt, but I don't think it's necessary. Or is there a case in which you think this would prevent a merge that otherwise would happen?